docs/source/asr/results.rst

```diff
@@ -63,6 +63,19 @@ If you would like to programatically list the models available for a particular
   nemo_asr.models.<MODEL_BASE_CLASS>.list_available_models()
 
 
+Transcribing/Inference
+^^^^^^^^^^^^^^^^^^^^^^
+
+You may perform inference and transcribe a sample of speech after loading the model by using its 'transcribe()' method:
+
+.. code-block:: python
+
+    model.transcribe(paths2audio_files=[list of audio files], batch_size=BATCH_SIZE, logprobs=False)
+
+Setting argument 'logprobs' to True would return the log probabilities instead of transcriptions. You may find more detail here: `transcribe() <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/r1.0.0rc1/asr/api.html#modules>`
```

Comment:
No hard  coded paths to docs. Use relative paths with rst. Look at how I do relative path to Jasper block in Citrinet.

Reply:
Done. Just need to test the docs.